src/components/general/NavigationDrawer/components/Grouping.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC, useMemo, useCallback } from 'react';
+import styles from './styles.less';
+import classNames from 'classnames';
+import { useComponentDisplayClassNames } from '@equinor/fusion';
+import { DropdownArrow } from '@equinor/fusion-components';
+import { getNavigationComponentForItem } from '../utils';
+import { NavigationComponentProps } from '..';
+
+const Grouping: FC<NavigationComponentProps> = ({
+    title,
+    id,
+    onClick,
+    isActive,
+    icon,
+    navigationChildren,
+    isOpen,
+    onChange,
+    isCollapsed,
+}) => {
+    const containerClassNames = classNames(
+        styles.container,
+        useComponentDisplayClassNames(styles),
+        {
+            [styles.menuGrouping]: true,
```

Comment:
Since this class is allways "on" it could be moved to inbetween styles.container and useComponentDisplayClassNames. E.g. styles.container, styles.menuGrouping, useComponentDisplayClassNames(...)